themes/default/content/blog/continue-on-error/index.md

```diff
@@ -0,0 +1,58 @@
+# yourself if you don't already have one.
+authors:
+    - thomas-gummerer
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - error-handling
+    - announcement
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md
+# for details, and please remove these comments before submitting for review.
+---
+
+This might sound like a familiar issue: you try to set up or destroy some cloud infrastructure and a resource has errors.  Pulumi now also errors out and leaves the resources it hasn't destroyed or updated yet untouched.  Often that's what you want. There might be no point in bringing up more infrastructure if a resource fails.  Or to destroy the rest of the stack if we can't destroy everything.
+
+In other cases though, it can be very useful to keep going trying to bring resources that are independent from the failed one into the requested state, be that resources being created or destroyed.  To allow you to do exactly this we're introducing a new `--continue-on-error` flag for both `pulumi up` and `pulumi destroy`.
+
+<!--more-->
+
+Using this flag means that resources that are not in the same dependency tree as the failed resource will still continue to be updated or destroyed, as they would normally do.  Without it Pulumi would stop and refuse to change any more resources after it encountered errors.  To make sure dependencies are still correctly respected, resources that depend on a successful update or destroy of the failed resource will not continue to be updated.  This means that this flag is always safe to use, as Pulumi will continue to manage the failed resources, and they can be updated or destroyed in subsequent runs of Pulumi.
+
+When the execution finishes, Pulumi will report the resource failures as you would currently expect from a failure, and exit with a non-zero exit code.  This indicates that even though we continued to update resources independent from the failed one, there was an error during the deployment.
+
+## What do I need to do?
+
+`pulumi destroy --continue-on-error` was introduced in Pulumi v3.112.0.  After upgrading to this version you can go ahead and use this feature.
+
+The situation for `pulumi up --continue-on-error` is slightly more complicated.  Since we now have resources that fail to create, but the pulumi program continues to execute, the SDK has to deal with the outputs of these resources.  Currently during `pulumi up` there are no "unknown" values expected by the SDK.  Since we may now have a failed resource, the outputs for that resource might still be unknown, and the SDK will have to deal with that.  This means that to fully support this feature some SDK changes were necessary.  So in addition to upgrading to Pulumi v3.114.0, you will also have to upgrade the Pulumi SDK to v3.114.0 for Go, Python and NodeJS, 3.63.0 for dotnet and 1.7.0 for yaml.
```

Comment:
```suggestion
`pulumi up --continue-on-error` was introduced in Pulumi v3.114.0. However, it also requires updated language SDKs to work best. Since we now have resources that fail to create, but the Pulumi program continues to execute, the SDK has to deal with the outputs of these resources.  Currently during `pulumi up` there are no "unknown" values expected by the SDK.  Since we may now have a failed resource, the outputs for that resource might still be unknown, and the SDK will have to deal with that.  This means that to fully support this feature some SDK changes were necessary.  So in addition to upgrading the Pulumi CLI to v3.114.0, you will also have to upgrade the Pulumi SDK to v3.114.0 for Go, Python and Node.js, 3.63.0 for .NET. For Pulumi YAML programs, you'll need to use v3.115.0 of the Pulumi CLI.
```